cpu/samd5x/cpu.c

```diff
@@ -102,12 +102,16 @@ static void xosc32k_init(void)
         return;
     }
 
+    /* Startup should be a valid value 0 - 6 (~63 ms to 8000 ms) see manual
+     * table 29-2 : 3 ^~= 1 sec
+     * this delay will happen only when the system is booted or the XOSC32
+     * is reenabled after beeing disabled for e.g.: standby*/
     OSC32KCTRL->XOSC32K.reg = OSC32KCTRL_XOSC32K_ENABLE
                             | OSC32KCTRL_XOSC32K_EN1K
                             | OSC32KCTRL_XOSC32K_EN32K
                             | OSC32KCTRL_XOSC32K_RUNSTDBY
                             | OSC32KCTRL_XOSC32K_XTALEN
-                            | OSC32KCTRL_XOSC32K_STARTUP(7);
+                            | OSC32KCTRL_XOSC32K_STARTUP(3);
```

Comment:
What's the rationale behind the change 7 -> 3?

Reply:
7 is not allowed 
6 would be 8 seconds boot 
3 is ~1 second  boot

Reply:
better add a comment that 3 means 500 ms then

Reply:
i did in line 107